plugin/storage/grpc/shared/extra/capabilities.go

```diff
@@ -0,0 +1,7 @@
+package extra
```

Comment:
this is used directly from `plugin/storage/grpc/shared/interface.go`. Is there a reason for having this in a separate package?

Reply:
This package was introduced to remove circular imports: `github.com/jaegertracing/jaeger/plugin/storage/grpc/shared` and `github.com/jaegertracing/jaeger/plugin/storage/grpc/shared/mocks`